uuv_dave/launch/uuv_dave.launch

```diff
@@ -71,14 +71,21 @@ UUV launch for project DAVE
         <arg name="yaw" value="-1.8"/>
     </include>
 
-    <!-- joystick control for rexrov and oberon7 -->
+    <!-- Velocity teleop (UUV stays in position when joystick is not used) -->
```

Comment:
Instead of commenting out, lets please add "if" and "unless" conditionals on an arg such as "suppress_stationary", ref. http://wiki.ros.org/roslaunch/XML

Reply:
OK, thanks. I added this.